specs/src/main.js

```diff
@@ -0,0 +1,95 @@
+const changeThemeSelect = document.getElementById('js-changeTheme');
+const customThemeStyleSheet = document.getElementById('js-customTheme');
+const localStorageSelectChoice = localStorage.getItem('selectChoice') || 0;
+const hoverClass = document.querySelectorAll('.js-hoverClass');
+const codeWrapperTabs = document.querySelectorAll('.code-wrapper-tab');
+
+// Store current CSS theme in localStorage
+
+customThemeStyleSheet.setAttribute(
+  'href',
+  changeThemeSelect.options[localStorageSelectChoice].value
+);
+
+if (changeThemeSelect) {
+  changeThemeSelect.value =
+    changeThemeSelect.options[localStorageSelectChoice].value;
+  changeThemeSelect.addEventListener('change', (e) => {
+    customThemeStyleSheet.setAttribute(
+      'href',
+      changeThemeSelect.options[e.target.options.selectedIndex].value
+    );
+    localStorage.setItem('selectChoice', e.target.options.selectedIndex);
+  });
+}
```

Comment:
the style is persisted in localStorage, and it displays it correctly in select, but I think the beginning of this block is wrong, and still chooses the default value.

comment is non-blocking as this is already the case since move to astro, probably earlier too